resources/views/project/comments.blade.php

```diff
@@ -0,0 +1,55 @@
+<h6>Comments</h6>
+
+<ul class="list-group">
+    @foreach($project->comments as $comment)
+        <li class="list-group-item">
+
+            <p>{{ $comment->comment_body }}</p>
+            <p class="small"><br>
```

Comment:
I could just imagine Dr J squinting at this and making a comment about how small it is haha